qa/integration/specs/logstash_to_logstash_spec.rb

```diff
@@ -36,13 +36,6 @@
     @fixture.teardown
   }
 
-  def change_logstash_setting(logstash_service, name, value)
```

Comment:
This stomps on the existing config in the logstash installation under test, which is shared across multiple tests and _may_ contain useful bits in the event of feature flags.

Since it is only ever used to set the `api.http.port` in advance of spawning the logstash process, we can simply use the equivalent `--api.http.port` command-line flag and avoid modifying the logstash config.